src/otx/algorithms/classification/adapters/openvino/task.py

```diff
@@ -126,7 +126,11 @@ def __init__(
 
         self.model = Model.create_model(model_adapter, "otx_classification", self.configuration, preload=True)
 
-        self.converter = ClassificationToAnnotationConverter(self.label_schema)
+        if self.model.hierarchical:
+            hierarchical_info = self.model.hierarchical_info["cls_heads_info"]
```

Comment:
Could we use the more specific name ?  
```suggestion
            hierarchical_cls_heads_info = self.model.hierarchical_info["cls_heads_info"]
```

Reply:
Fixed here and everythere, thanks